requirements-hdxckantool.txt

```diff
@@ -0,0 +1,2 @@
+beautifulsoup4==4.9.1
+awscli==1.19.112
```

Comment:
In the end is this a version of aws cli that runs with our dependencies ?

Reply:
Yes, @alexandru-m-g; and `beautifulsoup4` is the same version required by dev requirements so it doesn't break those either.